bigquery/README.md

```diff
@@ -4,3 +4,37 @@ This notifier pushes build data to a BigQuery instance.
 
 *Alpha Feature - Under Development*
 
+SQL queries through the BigQuery CLI:
+
+bq query '\<SQL QUERY\>'
+
+Legacy SQL dialect is set on default for the BigQuery CLI and must be disabled for the example queries to work.
+This can be done by editing ~/.bigqueryrc and adding the following lines:
+
+```
+[query]
+--use_legacy_sql=false
```

Comment:
In addition to mentioning this, you may want to link to these docs: https://cloud.google.com/bigquery/docs/reference/standard-sql/enabling-standard-sql